### ISSUE TITLE
Requests: Extend autoNormalize functionality

### ISSUE THREAD

**Issue by vicmarto** [syncthing/syncthing#5636]
Would be possible to extend the autoNormalize functionality to make the desired UTF-8 normalization form selectable? Thanks.

https://docs.syncthing.net/advanced/folder-autonormalize.html

**Comment by AudriusButkevicius** [comment]
Why?

**Comment by calmh** [comment]
Well, this would make sense whenever the expected normalization differs from the system default. The one scenario that comes to mind is an Mac with an NFS mount where we *don't* want to use the Mac's fucked up normalization.

**Comment by vicmarto** [comment]
Yeah, that's right. Sometimes the synched directory is exported via NFS and the clients gets and unexpected encoding.
Making it selectable would make Syncthing more versatile.

**Comment by calmh** [comment]
What I would do in that case is disable normalization in Syncthing and enforce it (if possible) on the NFS server, though.

**Comment by vicmarto** [comment]
Sorry, I don't think that would be possible for the NFS protocol.

**Comment by calmh** [comment]
Yeah, that depends. If the backing store is ZFS you set the `normalization` property and it happens at the filesystem level regardless of what the client wants, but not NFS in general I guess.

**Comment by calmh** [comment]
I'm tagging this as enhancement because it's possible and not terribly difficult to do, but it's far down on *my* to do list...

**Comment by vicmarto** [comment]
Thanks @calmh! Glad for your consideration.
